models/stg_zendesk__user.sql

```diff
@@ -37,6 +37,7 @@ final as (
         email,
         name,
         organization_id,
+        case when phone in ('-', 'null', 'NA') then null else phone end as phone,
```

Comment:
Request to keep formatting of case when consistent.
```suggestion
        case when phone in ('-', 'null', 'NA') 
            then null 
            else phone 
        end as phone,
```

Reply:
Additionally, this seems like a real data quality issue on the connector side. Would you be able to follow up in the data team channel to understand why we sync the `phone` records like this? Also, are they any other `null` string types that we need to consider here? (ie. `NaN`, etc.)

Reply:
Will do! And these were all the anomalous values that I saw in our internal Zendesk data at least, so it's unclear if other users would see different `null` string types.

Tried doing some digging from the Zendesk side... these [docs](https://support.zendesk.com/hc/en-us/articles/4408821302810-Understanding-how-Talk-calls-become-tickets#topic_hz3_5nm_sx) seem to suggest that the `phone` field is automatically populated by the system, but that internal users can also manually add phone numbers to profiles 🤔